src/woeusb

```diff
@@ -1,2060 +1,9 @@
-#!/usr/bin/env bash
-# A Linux program to create bootable Windows USB stick from a real Windows DVD or an image
-# Copyright © 2013 Colin GILLE / congelli501
-# Copyright © 2017 slacka et. al.
-#
-# This file is part of WoeUSB.
-#
-# WoeUSB is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
-#
-# WoeUSB is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with WoeUSB  If not, see <http://www.gnu.org/licenses/>.
-#
-# Notes:
-# * Use --strip instead of --no-symlinks for compatibility of Ubuntu 14.04(EoL: April 2019)
-#
-# We use indirections and primitive variables, which is false positive of this rule
-# shellcheck disable=SC2034
+#!/usr/bin/python3
 
-## Makes debuggers' life easier - Unofficial Bash Strict Mode
-## BASHDOC: Shell Builtin Commands - Modifying Shell Behavior - The Set Builtin
-set -o errexit
-set -o errtrace
-set -o nounset
-set -o pipefail
+import sys
 
-## Enable aliases for easy access to functions
-shopt -s expand_aliases
+sys.path.append("/opt/woeusb")
```

Comment:
I didn't know where put to files, so I just assumed they will be in /opt/woeusb

Reply:
Git Cola uses a mechanism to guess the proper location of the installation:
https://github.com/git-cola/git-cola/blob/master/bin/git-cola

This mechanism allows the application to be directly run from the source tree (i.e. without installation)

Reply:
Also a few environmental variables help applications to be *relocatable*, like `XDG_DATA_DIRS` etc., the freedesktop specs can help.

Reply:
If I'm not wrong, prefix stores a folder one higher than the one from which the script is run, and then using this, script seek ../cola/__init__.py

Reply:
so, if run from /usr/bin script will seek /usr/cola

Reply:
So this is what we want, or I just missing something?